tests/partitioner/test_argument_validation.py

```diff
@@ -0,0 +1,62 @@
+"""Tests of argument validation, in the absense of command line parsing"""
+
+
+import data_paths as dc
+import pytest
+
+from partitioner.arguments import PartitionArguments
+
+
+def test_none():
```

Comment:
see first comment, these should probably be in the `unittest` framework.

if you want, I could take on the task of switching those tests over (shouldn't take too long).

also if you want, we could just stick with pytest framework since we should be supporting it anyway 🤷‍♀️